### DISCUSSION TITLE
I do not have the /opt/so directory

### DISCUSSION THREAD

**Discussion by veraxcyber** [Security-Onion-Solutions/securityonion#4090] [Unsupported Versions] [answered]
Hello, 

After installation, I do not have the /opt/so directory path to be able to see any of the following:

Could anyone please assist? 

Setup /root/sosetup.log
Suricata /opt/so/log/suricata/suricata.log
Elasticsearch /opt/so/log/elasticsearch/<hostname>.log
Kibana /opt/so/log/kibana/kibana.log
Logstash /opt/so/log/logstash/logstash.log
Elastalert /opt/so/log/elastalert/elastalert_stderr.log

**Comment by TOoSmOotH** [comment]
You need to use sudo to see those files. sudo su -

**Reply by veraxcyber** [reply]
Hello, thanks for the reply. I still don't seem to have them. I selected Suricata on setup but don't seem to have any of the log files anywhere. Has it setup Snort with the Suricata engine rather than the Suricata IDS? Unsure why I cannot find the Suricata eve.JSON file.

**Comment by dougburks** [comment, marked as answer]
Is it possible you are running the old Security Onion 16.04?  Make sure that you're running the latest Security Onion 2.3.50:
https://blog.securityonion.net/2021/04/security-onion-2350-now-available.html

**Reply by veraxcyber** [reply]
I think I am on 16.04. With the IDS, I selected Suricata on the setup phase but it looks like I have Snort but with the Suricata signatures, is this why I would not be able to see any of the Suricata setup files such as eve.JSON? Many Thanks

**Reply by gcsrellis** [reply]
On 16.4 you can configure Suricata via /etc/nsm/HOSTNAME-INTERFACE/suricata.yaml (where HOSTNAME is your actual hostname and INTERFACE is your actual sniffing interface).

**Reply by dougburks** [reply]
Security Onion 16.04 has reached End Of Life and is no longer supported:
https://blog.securityonion.net/2021/04/security-onion-1604-has-reached-end-of.html

Please use the new Security Onion 2.3.50:
https://blog.securityonion.net/2021/04/security-onion-2350-now-available.html

**Reply by veraxcyber** [reply]
Thank you all, is it possible to use the soup command to upgrade to this new version?

**Reply by veraxcyber** [reply]
Hey gcsrellis, do you know where the eve.JSON file is located? Many Thanks

**Reply by gcsrellis** [reply]
I haven't ran 16.04 in a while but if I remember correctly it is referenced in the suricata.yaml file.  best of luck!

**Reply by veraxcyber** [reply]
Thank you, I found it but I am not sure where it outputs to. Appreciate your help!

**Reply by dougburks** [reply]
> Thank you all, is it possible to use the soup command to upgrade to this new version?

The 16.04 EOL article that I linked to earlier (https://blog.securityonion.net/2021/04/security-onion-1604-has-reached-end-of.html) says the following:

If you still have existing installations of Security Onion 16.04, please upgrade to Security Onion 2:
https://docs.securityonion.net/en/2.3/appendix.html
